Add more spaceport pictures

roadrunner56 opened the pull request:
## Summary
This PR is in response to issue #6446. It aims to replace duplicate spaceport images with new ones. 
I'm marking this as a draft as I've never attempted to make a PR of this scale before. Please bear with me if there are technical issues or things I missed.
I used Unsplash and Wikimedia Commons as a source of photographs. For the stations, I tried using more abstract images. Suggestions for better pictures are more than welcome. 
Happy New Year!

**Update**
I think I added the copyright attributions correctly, but I'm not sure. If anyone more familiar with copyright could give it a lookover, I'd appreciate it.
The High DPI PR is here https://github.com/endless-sky/endless-sky-high-dpi/pull/206

Arachi-Lover commented:
Most of the new Coalition ones look pretty good, though I'm still not really feeling the new Ahr one.
I'd like to keep the forest look to it, so maybe the other world that uses `forest4` (Osaeli) could have it replaced with the new one?
Alternatively could lean into the rainforest look, something like the new Midway Emerald image.

The other one I'm not really a fan of is the new Ring of Friendship one, it just looks... confusing? I'm not sure what I'm looking at in that one.

Amazinite reviewed (commented):
Haven't looked at which planets are getting which images, just looking over the images:

station42 is very clearly the inside of an abandoned warehouse and station29 is the inside of an abandoned train car. They don't fit well as station images, especially station29 where you can see outside the train car. I'm not too sure on station25 either.

Not too sure about how close up forest6, 7, 8 are. Landscape images are typically from farther back.

Otherwise these look great. Do you have 2x versions of all these?

roadrunner56 commented:
> Most of the new Coalition ones look pretty good, though I'm still not really feeling the new Ahr one. I'd like to keep the forest look to it, so maybe the other world that uses `forest4` (Osaeli) could have it replaced with the new one? Alternatively could lean into the rainforest look, something like the new Midway Emerald image.
> 
> The other one I'm not really a fan of is the new Ring of Friendship one, it just looks... confusing? I'm not sure what I'm looking at in that one.

I changed the images for those spaceports, are they alright now?

Arachi-Lover commented:
The one for Ring of Friendship is still a bit odd, but it's better.

Ahr seems to still have the `fields15` image?

roadrunner56 commented:
> Haven't looked at which planets are getting which images, just looking over the images:
> 
> station42 is very clearly the inside of an abandoned warehouse and station29 is the inside of an abandoned train car. They don't fit well as station images, especially station29 where you can see outside the train car. I'm not too sure on station25 either.
> 
> Not too sure about how close up forest6, 7, 8 are. Landscape images are typically from farther back.
> 
> Otherwise these look great. Do you have 2x versions of all these?


Stations 25, 29, and 42 are meant to be used for the Korath stations, but I had a hard time finding images that could be passed off as an abandoned space station. Do you have any idea where I could find better images?
I changed the photos for forest6, 7, and 8, do they work better as landscape images?

roadrunner56 commented:
> The one for Ring of Friendship is still a bit odd, but it's better.
> 
> Ahr seems to still have the `fields15` image?

Sorry, I think it should be fixed now?

roadrunner56 commented:
> Otherwise these look great. Do you have 2x versions of all these?

I'm... not really sure what that means? Should those images have a 1440x720 resolution? In which folder would I add them to?
This is for the high DPI plugin, right?
I'm really sorry, I might have bit off more than I could chew.

roadrunner56 commented:
> Stations 25, 29, and 42 are meant to be used for the Korath stations, but I had a hard time finding images that could be passed off as an abandoned space station. Do you have any idea where I could find better images?

I actually managed to find some images that sort of worked, are they ok now?

roadrunner56 commented:
>Otherwise these look great. Do you have 2x versions of all these?

I'm holding off on making a PR to the high DPI repo until people confirm that they're happy with the images chosen. Afterwards, I'll make a high DPI PR with the images.

Amazinite commented:
> I actually managed to find some images that sort of worked, are they ok now?

Those new three station images are better. Not too sure on all the abstract images working across all station images, though. Many of the stations that use duplicate landscapes are Korath ones, of which these abstract images don't seem like they'd fit.

roadrunner56 commented:
> Those new three station images are better. Not too sure on all the abstract images working across all station images, though. Many of the stations that use duplicate landscapes are Korath ones, of which these abstract images don't seem like they'd fit.

Can you point to any pictures specifically that you don't think work?

Amazinite commented on data/map.txt:
```diff
@@ -27195,7 +27195,7 @@ planet "Fek Rembatu"
 planet "Fenrir Station"
 	attributes deep station
-	landscape land/sivael1
+	landscape land/station4
```
Station4 fell out of use because it includes people in it, which goes against the landscape image guidelines. We might as well replace it.

roadrunner56 replied:
I tried replacing it with something else. Is the new image alright?

Amazinite commented on data/map.txt:
```diff
@@ -27831,7 +27831,7 @@ planet Janiculum
 planet Kaeyin
 	attributes uninhabited
-	landscape land/myrabella5
+	landscape land/station13
```
This planet isn't a station.

Amazinite commented on data/map.txt:
```diff
@@ -27875,7 +27875,7 @@ planet "Ki Patek Ka"
 planet "Korati Efreti"
 	attributes efret station
-	landscape land/loc2
+	landscape land/station15
```
I'm not too sure on the use of station15 here. The new image looks rather clean, but the old image gave more the impression of a dirty old station that I imagine better fits the reality of the state of Korath stations.

roadrunner56 replied:
Tried changing the image here, does it work?

Amazinite commented on data/map.txt:
```diff
@@ -27907,7 +27907,7 @@ planet "Kort Vek'kri"
 planet "Kraken Station"
 	attributes "near earth" station
-	landscape land/sivael7
+	landscape land/station8
```
Kraken Station is described as being very old. sivael7 fits more of the old station vibe than station8.

Amazinite commented on data/map.txt:
```diff
@@ -29506,20 +29506,20 @@ planet "Tebuteb's Table"
 planet "Tefkar Ret"
 	attributes korath station
-	landscape land/station0
+	landscape land/station12
```
The description says that this station is run down and abandoned, but station12 doesn't look very run down compared to station0.

Amazinite commented on data/map.txt:
```diff
@@ -29656,7 +29656,7 @@ planet Trove
 planet Tundra
 	attributes "dirt belt" oil textiles
-	landscape land/dmottl1
+	landscape land/snow20
```
I'd rather Tundra keep dmottl1 and we change whatever other planet has dmottl1 to snow20.

Amazinite commented on data/map.txt:
```diff
@@ -27846,7 +27846,7 @@ planet Janiculum
 planet Kaeyin
 	attributes uninhabited
-	landscape land/myrabella5
+	landscape land/city19
```
When I said this isn't a station I didn't mean just change the image name to something other than station. Planets should have proper landscape images while stations have interior images, but this is a planet with an interior image.

Amazinite commented on data/map.txt:
```diff
@@ -26512,7 +26512,7 @@ planet "Brass Second"
 planet "Bright Echo"
 	attributes folklore saryd tourism urban
-	landscape land/dmottl1
+	landscape land/snow20
```
Hmm, snow20 doesn't really fit here either. It's a good image so I'd keep it, but maybe find something less lifeless since this is described as an urban world that's only slightly colder than the Saryd homeworld.

roadrunner56 commented:
I just noticed that station15 and station 4 were under non-compatible licenses, so I changed them, as well as updating the copyright file and taking into account the feedback above. I replaced station4 with the original city19 image, and replaced station15 with a new image from Flickr. I'm not entirely sure how copyright formatting works for Flickr images, so apologies if I've messed something up.

Amazinite reviewed (approved):
Everything looks good to me now. Let's get that high DPI PR up.

roadrunner56 commented:
Because I lost all the original images, I had to redownload them and recrop them in order to maintain consistency with the 2x PR. Most of the images have not been substantially changed. I changed some images that on second thought I didn't feel were a good fit. These images are desert11, station20, station25, station27, station28, station30, and station36.

Amazinite reviewed (commented):
Not sure on the crops for station9, station13, and station15.
For station9, there's no issue with showing the floor, and it seems weird to me to crop it out.
For station13, I prefer the more upward looking view, whereas the new crop is basically just staring straight at the stairs.
For station15, I see that there's a person off in the distance and I imagine that's why you cropped it, but since they're so far out there and small in the frame it might be better to try and edit them out than crop the image to stare at the ceiling.

roadrunner56 commented:
Done.

Amazinite commented on copyright:
```diff
@@ -605,12 +695,35 @@ Files:
  images/land/station1*
  images/land/station2*
  images/land/station3*
- images/land/station4*
```
Looks like station4 was removed from the NASA copyright but wasn't readded anywhere else.

Wedge009 commented:
> For station9, there's no issue with showing the floor, and it seems weird to me to crop it out.

I just noticed this one while playing v0.9.16.1, it's the Wynyard Walk pedestrian tunnel in Sydney - it's very recognisable from using it every time I head in to the office. A bit immersion-breaking, but nice to know it looks 'high tech' enough to stand in for a space station (it's a fairly recent construction). (:
